src/views/Home.jsx

```diff
@@ -90,33 +93,58 @@ export function Home({ setListToken }) {
 	};
 
 	return (
-		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
+		<div className="home">
+			<p>Welcome to</p>
+			<h1>App Title!</h1>
+			<div className="app-logo">
+				<img src={appImage} alt="app logo" />
+			</div>
+			<p className="app-summary">
+				Summary Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+				Adipisci possimus tempora a quo deleniti dolore.
```

Comment:
Wondering what we would replace this text with or if we'd need it at all? I noticed when accessing the app in my browser, you have to scroll a little bit to see the Learn More link, so if we took this out, that may help to solve that.
![Screen Cast 2023-03-01 at 4 10 04 PM](https://user-images.githubusercontent.com/107324271/222287714-9b4dc561-3c63-47b7-b95f-6483a2b6cd4a.gif)

Reply:
@sarahmjenkins Oh yep, I haven't replaced the text yet. It's supposed to be a quick summary for the app's functionality! Yeah, I wasn't paying attention to how it looked on the web. We'd have to do that with media queries for larger screens, but I think that's a different issue (I think issue 19? Not sure).